custom/basic-call/components/Prejoin/Intro.js

```diff
@@ -35,10 +35,12 @@ export const Intro = ({
   }, [room]);
 
   return (
-    <form onSubmit={(e) => {
-      e.preventDefault();
-      onJoin(roomName, owner, fetchToken);
-    }}>
+    <form
+      onSubmit={(e) => {
```

Comment:
this can be moved into `submitForm` named method. (Please avoid inline functions as much as possible as a general rule👍 )